### PR TITLE
[#14] [Integrate] As a user, I can see the survey list in Home screen

### PR DESCRIPTION
**Pull request by luongvo**
https://github.com/luongvo/flutter-survey/issues/14

## What happened 👀

After logged in, the user can see the Home screen with the survey list.
 
## Insight 📝

- If successful, bind the data into the survey list.
- If failed, show the error message
- The navigation indicator list (bullets) must be dynamic and based on the API response.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/16315358/177049302-e31482be-e443-4446-8ca6-d1c5941ddaec.mp4

### CODE REVIEW

**Review by markgravity — COMMENTED** [review]
The rest LGTM

File: lib/pages/home/home_header.dart [hunk 1]
```diff
@@ -38,4 +40,10 @@ class HomeHeader extends StatelessWidget {
       ],
     );
   }
+
+  String _composeCurrentDate() {
+    final now = DateTime.now();
```
[the remark this answers is not in the document]
Comment (luongvo) [reply lib/pages/home/home_header.dart:45]: @markgravity good point, refactored

**Comment by luongvo** [comment]
@markgravity resolved ✅

**Review by markgravity — APPROVED** [review]
LGTM 👍

**Pull request merged**